include/lsst/jointcal/Gtransfo.h

```diff
@@ -163,7 +171,7 @@ class GtransfoIdentity : public Gtransfo {
                 "GtransfoIdentity is the identity transformation: it cannot be fit to anything.");
     }
 
-    std::unique_ptr<Gtransfo> reduceCompo(const Gtransfo *right) const { return right->clone(); }
+    std::unique_ptr<Gtransfo> reduceCompo(Gtransfo const &right) const { return right.clone(); }
```

Comment:
So long as you're changing the API, can you change the name to something a bit more standard?

Reply:
And this method needs to be documented.